docs/slack-apps/tutorials/channel-check.md

```diff
@@ -0,0 +1,70 @@
+---
+id: channel-check
+title: Check for New Channels
+sidebar_label: Check for New Channels
+---
+
+It can be hard to keep up with all of the changes in your Slack workspace! This tutorial will walk you through how to build a simple tool in Dark that posts all new, public Slack channels created in your Slack workspace in the past 24 hours. 
+
+A full sample canvas for this tutorial is available [here](https://darklang.com/a/sample-slackchannelcheck).
+
+## Slack Configuration
+
+1. [Create a new app in Slack](https://api.slack.com/apps).
+2. Navigate to Features -> OAuth& Permissions, scroll down to scopes and add the following bot token scopes: `channels:manage`, `channels:read`, `chat:write`, and `chat:write.public`. This will allow our app to interact with our the channels (also known as conversations) and post messages about them. 
+
+![assets/slack-apps/tutorials/channelcheck/bottokenscopes.png](assets/slack-apps/tutorials/channelcheck/bottokenscopes.png)
+
+3. Install your app to the workspace (either via Settings -> Basic Information or Settings -> Install App). Since this app is being built for your workspace specifically, we will not need to set up full OAuth, like we would if we wanted to distribute this more publicly.
```

Comment:
For words that you're trying to direct the reader to like this, I'd suggest using **bold** or *italics* (probably **bold**). (There are many places in this doc that would benefit from this).